content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -101,25 +101,30 @@ is busy handling the request, the worker won't start handling the next request
 immediately. This process adds to the queue wait time of the next request and affects
 average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-### Latency for Istio 1.19
+### Latency for Istio 1.20
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
 proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2),
-the two proxies add about 1.31 ms and 1.58 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
+the two proxies add about 0.228 ms and 0.298 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
 We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/{{< source_branch_name >}}/perf/benchmark)
-for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 16 client connections, 2 proxy workers and mutual TLS enabled.
+for the `http/1.1` protocol, with a 1 kB payload at 1000 requests per second using 2,4,8,16,32,64 client connections, 2 proxy workers and mutual TLS enabled.
 
-{{< image width="90%"
-    link="latency_p90_fortio_with_jitter.svg"
+Note: Different hardware will give different values.
+
+<br><img width="90%" style="display: block; margin: auto;"
+    src="latency-p90-fortio-with-uniform.svg"
     alt="P90 latency vs client connections"
     caption="P90 latency vs client connections"
->}}
+/>
+<p><h2 style="text-align: center;"> P90 latency vs client connections </h2></p><br>
+
 
```

Comment:
./en/docs/ops/deployment/performance-and-scalability/index.md:120: MD012 Multiple consecutive blank lines

Further documentation is available for these failures:
 - MD012: https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md012---multiple-consecutive-blank-lines:
```suggestion

```